### ISSUE TITLE
sftp extension will no longer connect to remote server

### ISSUE THREAD

**Issue by junderwood36** [Natizyskunk/vscode-sftp#401]
Hi, can you help me please. Really love using your extension, but I have a problem.

My sftp.json configuration file used to work perfectly for several months. But now I cannot connect to the remote server.

When I right click on a local file and upload, "ALL configuration Authentication methods failed"  
(I can still connect using the connection settings as I have done for last few months using FileZilla, so I believe everything is as expected at the server end)

SFTP debug is set to true. and I see this in the output pane of VS code (View > Output > sftp)

All configured authentication methods failed

My Desktop info
 - windows 10
 - VSCode Version 1.80.0
 - SFTP Extension (by natizyskunk) Version v1.16.3
 - SFTP debug is set to true

**Comment by satiromarra** [comment]
@junderwood36 Have you tried to downgrade to version 1.16.1?

**Comment by junderwood36** [comment]
Thanks for the suggestion Satiromarra. I just tried v 1.16.1  but it didn't help.

**Comment by satiromarra** [comment]
@junderwood36 Does your sftp connection require a password or rsa key?
If your RSA key has a password, you must indicate `"passphrase": true` to show a prompt to input the password. 
You could share your `sftp.json` by hiding sensitive data.

**Comment by junderwood36** [comment]
Thanks @satiromarr,a previously (from sept 2022 to dec 2022) it all worked with no problem using this config
![image](https://github.com/Natizyskunk/vscode-sftp/assets/71841847/c81eb71f-fbe2-46f2-b78c-914b6c97297a)

**Comment by Natizyskunk** [comment]
@junderwood36 If donwgrading to 1.16.1 doesn't fix your problem, you can try to downgrade further to 1.16.0 or try the pre-1.16.2 version.
For the pre-1.16.2, you're gonna have to download it from github and install it manually. You can find more help [here](https://github.com/Natizyskunk/vscode-sftp#method-2-manual-update).

Please let us know if it help to fix your issue 🙂

**Comment by Natizyskunk** [comment]
Related issues : #363, #376, #383, #397, #398

**Comment by junderwood36** [comment]
Hi Natizyskunk,  VScode with SFTP extension (latest by Natizyskunk) is working again. hooray!!

I changed the FTP password at the hosting platform

to the exact same password, I have used since September 2022
(and this is the same password that curiously was still working with FileZilla)
Now, VScode with SFTP extension (latest by Natizyskunk) just started working again. hooray!!
So nothing wrong with your brilliant extension, just something weird with cPanel or the hosting.

Thanks for your help, and many thanks for maintaining this brilliant extension, you are a star!
